pose_format/utils/holistic.py

```diff
@@ -52,11 +54,14 @@ def process_holistic(frames: list, fps: float, w: int, h: int, kinect=None, prog
         results = holistic.process(frame)
 
         body_data, body_confidence = body_points(results.pose_landmarks, w, h, 33)
-        face_data, face_confidence = component_points(results.face_landmarks, w, h, 468)
+        face_data, face_confidence = component_points(results.face_landmarks, w, h, 478 if holistic_config['refine_face_landmarks'] else 468)
         lh_data, lh_confidence = component_points(results.left_hand_landmarks, w, h, 21)
         rh_data, rh_confidence = component_points(results.right_hand_landmarks, w, h, 21)
+        # TODO: add pose_world_landmarks
```

Comment:
They also use `pose_world_landmarks`, guess we can also add them? Should be simple.

Reply:
you can add it as an optional additional component, equal to the normal pose body landmarks components, just a different name. (however, it should probably not be rendered, unless in a 3D world, as it is not centered on the image, but on the real world)